app/components/docs.py

```diff
@@ -104,5 +106,5 @@ async def docs(interaction: discord.Interaction, section: str, page: str) -> Non
         section_path, page = "config/keybind/", "sequence"
 
     await interaction.response.send_message(
-        URL_TEMPLATE.format(section=section_path, page=page)
+        f"{message}\n{URL_TEMPLATE.format(section=section_path, page=page)}"
```

Comment:
the redundant newline when message is empty is fine right?

Reply:
Yep, Discord seems to strip whitespace correctly:
<img src="https://github.com/user-attachments/assets/f18e809c-fae0-43c9-b941-946a76a7e3c8" width="60%">